test/Microsoft.Azure.WebJobs.Host.UnitTests/Scale/DefaultHostProcessMonitorTests.cs

```diff
@@ -327,26 +340,27 @@ public void GetStatus_MultiProcess_HighMemory_ReturnsOverloaded()
             Assert.Equal("[HostMonitor] Host memory threshold exceeded (8500 >= 8000)", log.FormattedMessage);
         }
 
-        [Fact(Skip = "Failing on CI. Need to investigate.")]
```

Comment:
This was the failing test that I wanted to enable. For whatever reason the 3 child processes I'd spawn below would start up and run for a short period of time, but would then die for some reason. Not reproable when I actually RDP'ed, or ran locally. The mocking I'm enabling in this PR is superior anyways.